### DISCUSSION TITLE
Addons vs Modifiers

### DISCUSSION THREAD

**Discussion by echoi-gwh** [CMSgov/hospital-price-transparency#153] [Q&A] [answered]
Hello,

I am creating a CSV Tall Format MRF for my org. My org is a psychiatric/mental health services hospital that whose main payer is the Los Angeles County Department of Mental Health (DMH) so they essentially dictate all of our rates.

One of the ways they decide rates for "professional services" that are mental health or psychiatry based is with "profession" modifiers (e.g. HL modifier for a licensed clinician providing a primary service will have a different rate than an AJ/AH modifier for a student/intern/fellow providing the same primary service). DMH explicitly refers to this as a "modifier".
- Example: The codes 90791:HL vs 90791: AH with two different rates

The other way a service may result in additional fees is with an addon service ("addon" is DMH terminology). This may include things like spoken language translation services or sign language translation services. Sometimes it's even an addon service for "longer than allotted time" rate. I believe they technically treat these addons as explicit services, not necessarily "modifiers" even though they can't be billed without a "primary" service and they often can only addon onto specific services.

So I have a couple questions given this context:

1. Following the example of 90791 mentioned earlier, regarding rates dependent on the profession modifier, may we do the following:
    a. Create a single 90791 record with all the profession modifiers listed, then the Minimum and Maximum columns filled out for the range of potential rates depending on the profession with the additional payer-specific information/additional generic information field to explain further.
    b. Create a record for each profession modifier with description and the additional payer-specific information/additional generic information field.
    c. If not, how do we approach the scenario of profession modifiers?
2. In the case of Hospital Price Transparency, do addon services count as modifiers? And therefore treat it the same as the profession modifiers?

**Comment by RidenourK** [comment, marked as answer]
@echoi-gwh. Thank you for your questions. 

As indicated in the CY 2024 OPPS/ASC Final Rule, hospitals are required to include modifiers only when they are necessary to provide the additional context needed for the standard charges the hospital has established. It is unnecessary to include modifiers that do not impact or change the standard charges established by the hospital. Please refer to the discussion starting at [88 FR 82104](https://www.federalregister.gov/d/2023-24293/p-5023).

Beginning January 1, 2025, your hospital must include modifiers that may change the standard charge. Different payers may have different modifiers that apply to the hospital’s standard charges and the Data Dictionary allows some flexibility for encoding this standard charge information. One approach is to separately encode the modifiers and describe how the modifier affects the standard charge established by the hospital, as demonstrated in the “Examples” found on the Hospital Price Transparency – Data Dictionary GitHub repository at [https://github.com/CMSgov/hospital-price-transparency/tree/master/examples.](https://github.com/CMSgov/hospital-price-transparency/tree/master/examples) In the examples, the modifier ‘50’ for a bilateral procedure negotiated by the Platform Health Insurance PPO plan applies to procedures performed in both the inpatient and outpatient settings, and increases the standard charge established between the hospital and payer’s plan by 150%. The ‘additional notes’ are used to explain that the payment adjustment that applies to the procedure’s ‘standard charge’ (which would be noted elsewhere in the file) is 150% when the modifier is appended to the claim. The modifier and the description of how it affects the standard charge are encoded on their own row in CSV MRFs and in a separate object in JSON MRFs.

Another approach would be to encode all combinations of codes, including modifiers, for each standard charge established. 

Examples of how to encode modifiers in the template can be found [here](https://github.com/CMSgov/hospital-price-transparency/tree/master/examples).
